client/components/social-logo/README.md

```diff
@@ -1,12 +1,12 @@
 Social Logo
 ========
 
-Component that renders a single socail-logo svg based on an `icon` prop. It takes a size property but defaults to 24px. For greater sharpness, the icons should only be shown at either 18, 24, 36 or 48px. There's a gallery with all the available icons in devdocs/design.
```

Comment:
Could we maybe add a link to https://github.com/Automattic/social-logos in the README? I wouldn't know we are speaking about that package if I didn't see this PR.

Maybe just adding a link to the text: "...renders a single [social-logo](link here) svg based...".